UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/Bodypart/NaturalChemicalReleaseComponent.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using HealthV2;
+using HealthV2.Living.PolymorphicSystems;
+using HealthV2.Living.PolymorphicSystems.Bodypart;
+using UnityEngine;
+
+public class NaturalChemicalReleaseComponent : BodyPartComponentBase
+{
+
+	/// <summary>
+	/// The Natural toxins that the body part makes ( give these to the liver to filter ) E.G Toxin
+	/// </summary>
+	[Tooltip("What reagent does this expel as waste?")]
+	public Chemistry.Reagent NaturalToxinReagent;
+
+	[Tooltip("How much natural toxicity does this body part generate Per tick per 1u of blood flow ")]
+	public float ToxinGeneration = 0.0002f;
+
+
+	[HideInInspector]
+	public ReagentCirculatedComponent reagentCirculatedComponent;
+
+	public float BloodThroughput
```

Comment:
this property seems to have the same implementation every time, could it be moved to the abstract base class?

Reply:
not everything needs blood throughput though?